tests/unit/output/test_local.py

```diff
@@ -48,6 +49,16 @@ def test_str_on_absolute_path(dvc):
     path = os.path.abspath(os.path.join("path", "to", "file"))
     output = OutputLOCAL(stage, path, cache=False)
 
+    assert path != str(output)
+    assert path == os.path.abspath(str(output))
+
+
+def test_base_str_on_absolute_path(dvc):
+    stage = Stage(dvc)
+
+    path = os.path.abspath(os.path.join("path", "to", "file"))
+    output = OutputBase(stage, path, cache=False)
+
```

Comment:
This is really odd that we test OutputBase here :slightly_smiling_face: Let's do:
```
output = OutputLOCAL(stage, path, cache=False)
assert output.def_path == path
```
where `path` is outside of dvc.root_dir.

Reply:
Also test name could  be adjusted to better reflect that abs path is out of repo.